### DISCUSSION TITLE
Where to place Agility in an organisational context in a antifragile world

### DISCUSSION THREAD

**Discussion by JRBliekendaal** [JRBliekendaal/master-thesis#18] [Q&A]
@edzob 

Working on my literature research (not standing still just alot of work in the background). Just wondering. In my one and five pager I defined fragile, resilience, robustness, agility and antifragility.

When I do read your thesis (Edzo, 2020) I miss agility. Did you also thought of how we should see Agility? Many organisations are transforming towards an agile organisation and they think that this will solve everything like a Number 42 (Douglas, 1979). While my opinion the better state is still antifragility.

The definitions I used are:

- Agility is the ability to adjust before failure happens
- Robustness is the ability to resist failure
- Resiliency can recover from failure.

This is btw not the economic viewpoint of value you used. So with these definitions i come to the order:

Fragile -> Resilience -> Robustness -> Agility -> Antifragile 

;) Not yet used the theories like those from Breen and Kastner (CAS).

**Comment by edzob** [comment]
Hi.

[1]
>> Agility is the ability to adjust before failure happens
I would double check this definition (there is a lot of research on this)

[2]
>>  before failure, resist failure, from failure
I would double check this definition. In the literature on stressor events
or x-events, or risk management (iso3100), or vuca
I think another definition is used.

[3]
When I discussed the EAAL with some experts they recognized many
developments/movements around agile in the EAAL model.
When you look at the attributes under CAS resilience and combine them with
the attributes of an learning organisation (5th discipline),
then you can reckognize many patterns of various agile movements.
That having said. I did not investigate this further, this was just an
observation of a few people who where looking from the context of
antifragility.

Does this help?

--
"It is amazing what you can accomplish if you do not care who gets the
credit." *-* Harry S Truman
*Signal*, Whatsapp, Telegram -
*+31629485425*
*Twitter*, Wire, Matrix Element - ***@***.****
*Threma* - *6FVWM3Z5*
*Homepage* - https://www.edzob.com/page/about
*ORCID* - *https://orcid.org/0000-0003-0097-7375
<https://orcid.org/0000-0003-0097-7375>*

**Reply by JRBliekendaal** [reply]
So you see Agile as some sort of container around other concepts.

To make sure I am conplete I will confront your thesis with agility to have a complete view on the definitions. I see with alot of ISV's are focussed on getting more Agile as an organisation. So when i do not take agility into account I think that Antifragility is less understood.

**Reply by edzob** [reply]
> So you see Agile as some sort of container around other concepts.
No.. I see agile as a specific subset of the superset created by union (CAS Resilience/Learning organisation)
And agile is specific in the domain of organisation where the EAAL is on organisations and more :)

**Comment by edzob** [comment]
in addition. If you want I can review the 1-pager and 5-pager and provide some feedback.

**Reply by JRBliekendaal** [reply]
I can send them to you.. but to be honest... they are rough idea's and I do not think that a lot of those text are coming over to the real thesis.

I did not have the time at that moment for a real five pager. So I decided with Hans just to have a presentation. Why no time? I have a fragile team in Lithuania :D Really have to make it antifragile. The Lithuanian organisation/department I build up is in her 3rd year anniversary. So a lot op people are leaving (including the knowledge) which is highly impacting the projects that are running from that department.

For more information.. I uploaded the concepts to the other repo. Nice to have an overview on the journey.
[Concepts](https://github.com/JRBliekendaal/master-thesis-administration/tree/main/concepts)
